extensions/wikia/AdEngine/js/context/adContext.js

```diff
@@ -75,11 +75,7 @@ define('ext.wikia.adEngine.adContext', [
 				context.opts.pageFairDetection = true;
 			}
 		}
-
-		// SourcePoint disaster recovery
-		if (w.wikiaSourcePointStatus === false) {
-			context.opts.sourcePointRecovery = false;
-		}
+		
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Remove the useless trailing whitespaces at the end of this line. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://10.8.74.32:9000/coding_rules#rule_key=javascript%3ATrailingWhitespace)